index.html

```diff
@@ -2,18 +2,16 @@
 <html>
 
     <head>
-        <title>Login - Spotify</title>
         <link rel="stylesheet" type="text/css" href="style.css" />
+        <title>Login - Spotify</title>
     </head>
 
     <body> 
 
         <div class="topnav">
-            <div class="navlogo">
                 <a href="https://open.spotify.com/">
-                    <img src="/images/spotify_logo_b&w.png" class="spotify_bandw_logo">
+                    <img class="spotify_bandw_logo" src="/images/spotify_logo_b&w.png" >
```

Comment:
I think we spoke about this, and I misinformed you, classes should always go before the `src` or `href` as they are more likely to be worked with in the future than links and its kind of annoying to have to scroll to the end to find a class.